<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>72</y>
      <width>228</width>
      <height>329</height>
     </rect>
    </property>
    <property name="title">
     <string>Random</string>
    </property>
    <widget class="QListWidget" name="numWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>72</y>
      <width>229</width>
      <height>329</height>
     </rect>
    </property>
    <property name="title">
     <string>Bubble Sort</string>
    </property>
    <widget class="QListWidget" name="bubbleWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>72</y>
      <width>228</width>
      <height>329</height>
     </rect>
    </property>
    <property name="title">
     <string>Qucik Sort</string>
    </property>
    <widget class="QListWidget" name="quickWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
